components/log_wrappers/src/lib.rs

```diff
@@ -86,7 +102,119 @@ impl<'a> slog::Value for Key<'a> {
         key: slog::Key,
         serializer: &mut dyn slog::Serializer,
     ) -> slog::Result {
-        serializer.emit_arguments(key, &format_args!("{}", hex::encode_upper(self.0)))
+        if REDACT_INFO_LOG.load(Ordering::SeqCst) {
```

Comment:
How about `Ordering::Relaxed`? As it actually does not require sequentially consistent.

Reply:
will update